src/components/HandleSocialAuth.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { socialAuth } from '../state/auth/action';
+
+
+export const handleSocialAuth = props => {
```

Comment:
Can you make `handleSocialAuth' PascalCase, since it's a component?

Reply:
thanks for your feedback, done. Thanks